### PR TITLE
feat(faq): connect bot to ElasticSearch

### LINKED ISSUE #271: Call Rasa the app to get the answer
Please sync up with @lp-noel

### PR DESCRIPTION
**Pull request by vicrep**
Closes #271 , #272 

As mentioned [below](https://github.com/dialoguemd/covid-19/pull/383#pullrequestreview-380597190), waiting on model improvements to merge.

### CODE REVIEW

**Comment by mbelang** [comment]
Well we do not have rasa so the PR title is not good :P

**Comment by vicrep** [comment]
I was copying title from issue @mbelang , better now? 😆

File: src/services/faq.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export const getAnswers = async (question: string, language?: string) => {
+  const headers: Record<string, string> = {}
+
+  if (language) {
+    headers['Accept-Language'] = language
+  }
+
+  const res = await faqApi.get<GetAnswersResponse>(
+    '/answers/',
```
Comment (mbelang) [inline src/services/faq.ts]:
```suggestion
    '/answers',
```
I fixed the backend

File: src/services/faq.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import api from 'apisauce'
+
+const URL = 'https://covidfaq.apps.dev.ca-central-1.dialoguecorp.com' // FIXME: through config?
```
Comment (lp-noel) [inline src/services/faq.ts]: this can be changed to: `https://covidfaq.dialoguecorp.com`

**Comment by lp-noel** [comment]
@vicrep will we get tracking in analytics of the feedback given by people to the answers chloe gives to their question? e.g. If a user asks 'what are the symptoms', we reply 'fever and cough' and the user clicks on 'completely off', do we get all three of these values in analytics DB?

**Comment by vicrep** [comment]
@lp-noel you get tracking of the question asked, the rating, but not the answer

**Comment by lp-noel** [comment]
> @lp-noel you get tracking of the question asked, the rating, but not the answer

@vicrep can we implement that? Or will we be able to link these from the DB with an id or something?

**Comment by vicrep** [comment]
I'm not sure if you'd be able to derive from existing events, we might need to start tracking them -- but then, wouldn't you want a answer ID instead of a plaintext representation of answer?

**Comment by lp-noel** [comment]
Yes it could be an answer id, makes sense. As long as we are able to make the link between question, answer and rating.

**Pull request merged**